buildenvs/gcc.Dockerfile

```diff
@@ -130,10 +101,11 @@ RUN set -ex; \
     esac; \
     GCC_CONFIGURE_ARGS="$GCC_CONFIGURE_ARGS --target=${GCC_PREFIX}"; \
     ../gcc-${GCC_VERSION}/configure ${GCC_CONFIGURE_ARGS}; \
-    make all-gcc; \
+    make -j$(($(nproc) - 1)) all-gcc; \
```

Comment:
This is a bit dangerous to execute since it will consume all the resources of a machine. Perhaps use an ARG instead, e.g. `MAKE_NPROC` so this can be turned off, it's default value should be `1`, so:
```diff
+ ARG MAKE_NPROC=1
[...]
- make all-gcc; \
+ make -j${MAKE_NPROC} all-gcc; \
```